engines/python/setup/djl_python/rolling_batch/lmi_dist_rolling_batch.py

```diff
@@ -154,28 +154,25 @@ def translate_lmi_dist_params(self, parameters: dict):
         :return: The same parameters dict, but with lmi-dist style parameter names.
         """
         parameters["max_tokens"] = parameters.pop("max_new_tokens", 30)
-        # If `do_sample` is not provided, force temperature=0.0, i.e. greedy
-        # else set to user-provided value or default to 1.0
-        if not parameters.pop('do_sample', False):
-            parameters['temperature'] = 0.0
-        else:
-            parameters['temperature'] = parameters.get('temperature', 1.0)
-        if "seed" in parameters.keys():
+        # if do_sample is provided and temperature is not, default to 1.0
+        if parameters.pop('do_sample', False):
+            parameters['temperature'] = parameters.get("temperature", 1.0)
```

Comment:
Shouldn't we default to 0.0 for greedy?

Reply:
oh yes, good catch. If do_sample == false, then temp = 0. will update

Reply:
updated